src/main/java/edu/cornell/kfs/concur/businessobjects/ConcurEventNotification.java

```diff
@@ -105,4 +111,21 @@ public void setNotificationURI(String notificationURI) {
         this.notificationURI = notificationURI;
     }
 
+    public String toString() {
+        SimpleDateFormat sdf = new SimpleDateFormat(KFSConstants.MONTH_DAY_YEAR_DATE_FORMAT, Locale.US);
```

Comment:
Is it necessary to pass in the locale or could this use the default locale?

Reply:
I will verify in my local env that locale is not needed.